src/server_manager/ui_components/outline-server-settings.html

```diff
@@ -122,7 +123,7 @@
           <div>
             <h3>DigitalOcean</h3>
             <paper-input readonly value="[[serverLocation]]" label="[[localize('settings-server-location')]]" hidden$="[[!serverLocation]]" always-float-label maxlength="100"></paper-input>
-            <paper-input readonly value="[[serverMonthlyCost]] USD/month" label="[[localize('settings-server-cost')]]" hidden$="[[!serverMonthlyCost]]" always-float-label maxlength="100"></paper-input>
+            <paper-input readonly value="[[serverMonthlyCost]] USD" label="[[localize('settings-server-cost')]]" hidden$="[[!serverMonthlyCost]]" always-float-label maxlength="100"></paper-input>
```

Comment:
Maybe we should localize "per month"?

Reply:
I was thinking about it. Do you think it would be confusing not to localize it, since we already say "Montly cost" in the label?